python/ql/src/Security/CWE-079/Jinja2WithoutEscaping.qhelp

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+
+Cross-site scripting attacks can occur if untrusted input is not escaped. This applies to templates as well as code.
+The <code>jinja2</code> templates may be vulnerable to XSS if the environment has <code>autoescape</code> set to <code>False</code>.
+Unfortunately, <code>jinja2</code> sets <code>autoescape</code> to <code>False</code> by default.
+Explicitly setting <code>autoescape</code> to <code>True</code> when creating an <code>Environment</code> object will prevent this.
+</p>
+</overview>
+
+<recommendation>
+<p>
+Avoid setting jinja2 autoescape to False.
+Jinja2 provides the function <code>select_autoescape</code> to make sure that the correct auto-escaping is chosen.
+For example, it can be used when creating an environment <code>Environment(autoescape=select_autoescape(['html', 'xml'])</code>
+</p>
+</recommendation>
+
+<example>
+<p>
+The following example is a minimal flask app which shows a safe and unsafe way to render the given name back to the page.
+The first view is unsafe as <code>first_name</code> is not escaped, leaving the page vulnerable to cross-site scripting attacks.
+The second view is safe as <code>first_name</code> is escaped, so it is not vulnerable to cross-site scripting attacks.
+</p>
+<sample src="examples/jinja2.py" />
+</example>
+
+<references>
+<li>
+http://jinja.pocoo.org/docs/2.10/api/
+Jinja2: <a href="http://jinja.pocoo.org/docs/2.10/api/">API</a>.
```

Comment:
Suggest:
`<li>Jinja2: <a href="http://jinja.pocoo.org/docs/2.10/api/">API</a>.</li>`

To match the style of other references.